sklearn/metrics/_plot/tests/test_plot_confusion_matrix.py

```diff
@@ -264,3 +263,31 @@ def test_confusion_matrix_text_format(pyplot, data, y_pred, n_classes,
     text_text = np.array([
         t.get_text() for t in disp.text_.ravel()])
     assert_array_equal(expected_text, text_text)
+
+
+@pytest.mark.parametrize("values_format", [None])
+def test_confusion_matrix_standard_format(pyplot, data, y_pred, n_classes,
```

Comment:
I think the test would be more helpful if it were checking explicit formatted values.

i.e. pass in 1123124, make sure it's rendered as 1123124. Pass in .345, make sure it's rendered as .35...

It will make it easier for us to verify that it works as expected

Reply:
I am confused by what this latest change was trying to do. I liked how you had `assert_array_equal(expected_text, text_text)`, but I didn't like how expected_text was constructed by formatting the numbers, rather than as a list of strings manually entered into the test.

Reply:
I am confused by what this latest change was trying to do. I liked how you had `assert_array_equal(expected_text, text_text)`, but I didn't like how expected_text was constructed by formatting the numbers, rather than as a list of strings manually entered into the test.

Reply:
Ahh ok, I see. Sorry I misunderstood it and thought you meant just directly check these values formatting.